lib/cortex/client.rb

```diff
@@ -20,7 +20,7 @@ class Client
     include Cortex::Request
 
     def initialize(hasharg)
-      @base_url = hasharg[:base_url] || 'https://cbcortex.com/api/v1'
+      @base_url = hasharg[:base_url] || 'http://cortex.dev/api/v1'
```

Comment:
In this case, the original URL was invalid - it doesn't actually resolve to anything